lib/index.js

```diff
@@ -19,6 +20,12 @@ var KISSmetrics = module.exports = integration('KISSmetrics')
   .retries(2);
 
 /**
+ * Use a POST request if the GET query string payload exceeds 2KB.
+ */
+
+var MAX_GET_PAYLOAD = 1 << 11;
```

Comment:
`1<<11` is the same as `2^11` (`Math.pow(2, 11)`). To me `2<<10` looks like `2^10` which is 1024. Happy to change it if you prefer this format, or I can use a literal 2048 if you like 😛